src/settings.py

```diff
@@ -0,0 +1,206 @@
+"""
+Django settings for src project.
+
+Generated by 'django-admin startproject' using Django 2.2.16.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+import dj_database_url
+import dotenv
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+dotenv_file = os.path.join(BASE_DIR, ".env")
+if os.path.isfile(dotenv_file):
+    dotenv.load_dotenv(dotenv_file)
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('SECRET_KEY')
+
+ADMIN_PATH = os.environ.get('ADMIN_PATH')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = 'RENDER' not in os.environ
+
+ALLOWED_HOSTS = ['127.0.0.1', 'cloud.f4mp.net']
```

Comment:
Consider only allowing localhost as the allowed host when DEBUG is enabled as well as dissalowing `cloud.f4mp.net`.

Reply:
The app doesn't give any "superpowers" if the host http header is set to localhost or 127.0.0.1. If want to remove it then we'd have to add another env variable which I think isn't necessary at this point but if you want to change it?

Reply:
The would exist to limit the possibility of hosting with DEBUG enabled in production. It could be done by modifying the variable depending on the value of DEBUG.

Something like `ALLOWED_HOSTS = ("127.0.0.1") if DEBUG else ("cloud.f4mp.net")` as a simple check.

Reply:
```python
if DEBUG:
    ALLOWED_HOSTS = ['127.0.0.1']
else:
    ALLOWED_HOSTS = ['cloud.f4mp.net']
```

Has been implemented

Reply:
Can we resolve?